vspace/src/main/java/edu/asu/diging/vspace/web/staff/DeleteSlideController.java

```diff
@@ -0,0 +1,50 @@
+package edu.asu.diging.vspace.web.staff;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import edu.asu.diging.vspace.core.exception.SlideDoesNotExistException;
+import edu.asu.diging.vspace.core.services.ISlideManager;
+
+@Controller
+public class DeleteSlideController {
+
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+
+    @Autowired
+    private ISlideManager slideManager;
+
+
+
+    @RequestMapping(value = "/staff/module/{id}/slide/{slideId}", method = RequestMethod.DELETE)
+    public ResponseEntity<String> deleteSlide(@PathVariable("id") String moduleId,
+            @PathVariable("slideId") String slideId) {
+        try {
+            slideManager.deleteSlideById(slideId, moduleId);
+        } catch (SlideDoesNotExistException slideDoesNotExistException) {
+            logger.error("Could not delete slide.", slideDoesNotExistException);
+            return new ResponseEntity<String>("Sorry, unable to delete the slide. The slide does not exist.",
+                    HttpStatus.NOT_FOUND);
+        }
+        return new ResponseEntity<String>("Ok", HttpStatus.OK);
+    }
+
+    @RequestMapping(value = "/staff/module/{id}/slide/{slideId}/sequences", method = RequestMethod.POST)
```

Comment:
this path implies it's returning a list of sequences of some kind, so either return a list of sequence ids or something or change the path. I'd opt for version one (returning a list of sequence ids (and possible sequence titles)) since this will allow to e.g. highlight the sequences the slide is contained or list them in another way.